Sage/carbon
# fix(alert): fix styling bug with close icon - FE-3416

fixes styling bug introduced to alert in v53.1.1

### Proposed behaviour

<img width="506" alt="Screenshot 2020-12-09 at 12 58 11" src="https://user-images.githubusercontent.com/56251247/101632778-42424000-3a1e-11eb-9a33-5a35df6b60a1.png">

### Current behaviour

<img width="481" alt="Screenshot 2020-12-09 at 12 58 26" src="https://user-images.githubusercontent.com/56251247/101632797-4a9a7b00-3a1e-11eb-806a-5b3c89dddd7c.png">

### Checklist



### Additional context
N/A

### Testing instructions
Navigate to Alert story and check that the close icon is back to where it was prior to v53.1.1
